lib/findAccessorMinMax.js

```diff
@@ -35,15 +34,16 @@ function findAccessorMinMax(gltf, accessor) {
         if (defined(bufferId) && defined(buffers) && buffers.hasOwnProperty(bufferId)) {
             var buffer = buffers[bufferId];
             var source = buffer.extras._pipeline.source;
+
             var count = accessor.count;
             var byteStride = getAccessorByteStride(accessor);
             var byteOffset = accessor.byteOffset + bufferView.byteOffset;
             var componentType = accessor.componentType;
-            var componentByteLength = byteLengthForComponentType(componentType);
 
             for (var i = 0; i < count; i++) {
+                var typedArray = ComponentDatatype.createArrayBufferView(componentType, source.buffer, byteOffset, numberOfComponents);
```

Comment:
> This should go outside the loop.

This cannot go outside the loop. We can't guarantee that an accessor is contiguous in the buffer, and we also can't assume that byteStride is a multiple of our component size.

> I think this would be pretty inefficient.

Why? We're just making a view of the existing data, not copying anything.